tasks/sanity-inspect-image.yaml

```diff
@@ -0,0 +1,40 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: sanity-inspect-image
+spec:
+  params:
+    - name: SOURCE_IMAGE
+      description: the fully qualified image name
+      default: quay.io/hongweiliu/skopeo-inspect-test:latest
+      type: string
+    - name: BASE
+      description: Inspecting base image for source image
+      default: 'false'
+      type: string
+  steps:
+  - name: inspect-image
+    image: quay.io/redhat-appstudio/hacbs-test:latest
+    script: |
+      IMAGE_INSPECT_FILE=$(workspaces.conftest-ws.path)/image_inspect.json
+      BASE_IMAGE_INSPECT_FILE=$(workspaces.conftest-ws.path)/base_image_inspect.json
+      if [ $(params.BASE) == 'true' ];then
+        echo "Inspecting base image for source image: $(params.SOURCE_IMAGE)"
+        BASE_IMAGE_INSPECT_FILE=$(workspaces.conftest-ws.path)/base_image_inspect.json
+        if [ -f "$BASE_IMAGE_INSPECT_FILE" ];then
+          echo "$BASE_IMAGE_INSPECT_FILE exists"
+        else
+          BASE_IMAGE="$(jq -r ".Labels.\"org.opencontainers.image.base.name\"" $IMAGE_INSPECT_FILE)"
```

Comment:
Aren't the `org.opencontainers.image.base.name` and `org.opencontainers.image.base.digest` items supposed to be under `annotations` in the skopeo raw output?

Reply:
Yes. I revise the code and use the logic as below, try to get base image from `annotations`, if not found, will try to get base image from `Labels`, if not found, it will fail. Does it make sense?
```
BASE_IMAGE="$(jq -r ".annotations.\"org.opencontainers.image.base.name\"" $IMAGE_RAW_MANIFEST)"
if [ $BASE_IMAGE == 'null' ]; then
  echo "Can not get base image info from 'annotations'"
  echo "Trying to get base image info from 'Labels'"
  BASE_IMAGE="$(jq -r ".Labels.\"org.opencontainers.image.base.name\"" $IMAGE_MANIFEST)"
  if [ $BASE_IMAGE == 'null' ]; then
    echo "Can not get base image info from 'Labels', please check the source image $(params.SOURCE_IMAGE)"
    exit 1
  fi
fi
```

Reply:
Yes, I think this looks like a good solution.